contrib/qpl-cmake/CMakeLists.txt

```diff
@@ -16,16 +15,18 @@ function(GetLibraryVersion _content _outputVar)
     SET(${_outputVar} ${CMAKE_MATCH_1} PARENT_SCOPE)
 endfunction()
 
-set (QPL_VERSION 1.2.0)
+set (QPL_VERSION 1.6.0)
 
 message(STATUS "Intel QPL version: ${QPL_VERSION}")
 
-# There are 5 source subdirectories under $QPL_SRC_DIR: isal, c_api, core-sw, middle-layer, c_api.
-# Generate 8 library targets: middle_layer_lib, isal, isal_asm, qplcore_px, qplcore_avx512, qplcore_sw_dispatcher, core_iaa, middle_layer_lib.
+# There are 5 source subdirectories under $QPL_SRC_DIR: c_api, core-iaa, core-sw, middle-layer and isal.
+# Generate 8 library targets: qpl_c_api, core_iaa, qplcore_px, qplcore_avx512, qplcore_sw_dispatcher, middle_layer_lib, isal and isal_asm,
+# which are then combined into static or shared qpl.
 # Output ch_contrib::qpl by linking with 8 library targets.
 
-# The qpl submodule comes with its own version of isal. It contains code which does not exist in upstream isal. It would be nice to link
-# only upstream isal (ch_contrib::isal) but at this point we can't.
+# Note, qpl submodule comes with its own version of isal that is not compatible with upstream isal (e.g., ch_contrib::isal).
```

Comment:
That is not good and should please be addressed upstream. Perhaps the standard isa-l can be used in the meantime - kindly check.

Suggest to keep sentence `It would be nice to link [...]`, otherwise this situation will be perpetuated.

Reply:
Sorry, I'm not following, perhaps I formulated the sentence incorrectly..

QPL at some point integrated an ISA-L copy and it was heavily modified to QPL needs, we do not have plans to either be compatible with upstream or update this copy, and now all the names were changed to have `qpl_` prefix, so it shouldn't create any issues with any other copies.

Reply:
Okay, in this case we can indeed remove `It would be nice to link [...]`. A comment a la `QPL at some point integrated an ISA-L copy and it was heavily modified to QPL needs, we do not have plans to either be compatible with upstream or update this copy.` in the CMakeLists.txt file would be helpful.